### DISCUSSION TITLE
Hybrid assembly: getting list of circular contigs.

### DISCUSSION THREAD

**Discussion by Somebodyatthdoor** [ablab/spades#1145] [Q&A]
Hi,

I 've been doing a hybrid assembly on some metagenomic data using hybridspades, and I was wondering if somewhere hidden in the spades output files there is any info on which contigs are predicted to be circular, like there is for other assemblers that use long reads (hicanu, hifiasm, flye etc.)? 

If there isn't, any suggestions for how to produce this info would be very much appreciated.

Cheers,
Laura

**Comment by asl** [comment]
Hello

You can check the assembly graph for this. For example, you could visualize the final assembly using Bandage-NG and see circular contigs.
